src/test/sql/sqlite_testrunner/sqlite_testrunner.cpp

```diff
@@ -129,26 +129,31 @@ void SQLiteTestRunner::SetUp() {
   }
 }
 
-std::vector<std::string> SQLiteTestRunner::queries() {
-  static std::vector<std::string> queries;
+std::vector<std::pair<size_t, std::string>> SQLiteTestRunner::queries() {
+  static std::vector<std::pair<size_t, std::string>> queries;
 
   if (!queries.empty()) return queries;
 
   std::ifstream file("resources/test_data/sqlite_testrunner_queries.sql");
   std::string query;
 
+  auto next_line = size_t{0};
```

Comment:
This should really be 1. The printouts are off by 1.

Reply:
> This should really be 1. The printouts are off by 1.

Just double checked, both my vi and IDE start counting at 1.

Reply:
It's incremented before it is used. That way, we can get away with just one ++, even in the case of continue

Reply:
Will add a comment

Reply:
In line 145, there is a -1 again..

Reply:
I fixed that before you wrote this, didn't I?

Reply:
I am not sure, I still get confused by the GitHub PR tool.